pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -779,6 +872,32 @@ class CreateVmModal extends React.Component {
     render() {
         const { nodeMaxMemory, nodeDevices, networks, osInfoList, loggedUser, providerName, storagePools, vms } = this.props;
         const validationFailed = this.state.validate && validateParams({ ...this.state, osInfoList });
+        let startVmCheckbox = (
+            <label className="checkbox-inline">
+                <input id="start-vm" type="checkbox"
+                    checked={this.state.startVm}
+                    disabled={this.state.unattendedInstallation}
+                    onChange={e => this.onValueChanged('startVm', e.target.checked)} />
+                {_("Immediately Start VM")}
+            </label>
+        );
+        if (this.state.unattendedInstallation) {
+            startVmCheckbox = (
+                <OverlayTrigger overlay={ <Tooltip id='virt-install-not-available-tooltip'>{ _("Setting the user passwords for unattended installation requires starting the VM when creating it") }</Tooltip> } placement='top'>
```

Comment:
`placement=left` would be better, now it is floating over nothing,
![Screenshot from 2019-10-01 10-45-04](https://user-images.githubusercontent.com/12330670/65947616-f3bc0d80-e438-11e9-9605-a7777c53ff44.png)

Reply:
Fixed.